.github/workflows/linux_builds.yml

```diff
@@ -5,19 +5,18 @@ on:
 # Global Settings
 env:
   # Used for the cache key. Add version suffix to force clean build.
-  GODOT_BASE_BRANCH: '4.0'
+  GODOT_BASE_BRANCH: 4.0
+  GODOT_CPP_BRANCH: 4.0
```

Comment:
@Repiteo FYI, this cleanup we made a while actually breaks things when the value ends with `.0`.
It seems like it's parsed as a float and converted to `4`.

I found out when backporting changes to `4.0` as it broke checking out the `4.0` branch on godot-cpp: https://github.com/godotengine/godot/actions/runs/14665876605/job/41160594191?pr=105749